pure-selenium/src/test/java/tests/AddMultipleItemsTest.java

```diff
@@ -20,8 +16,8 @@ public void testAddMultipleProductsToCart() {
         InventoryPage inventoryPage = new InventoryPage(driver);
         Assert.assertTrue(inventoryPage.isPageDisplayed(), "Inventory page should be visible after login");
 
-        inventoryPage.addBackpackToCart();
-        inventoryPage.addBikeLightToCart();
+        inventoryPage.addProductToCartByName("Sauce Labs Backpack");
+        inventoryPage.addProductToCartByName("Sauce Labs Bike Light");
```

Comment:
You can create a method that accepts multiple product names and adds them to the cart